packages/driver/cypress/integration/commands/assertions_spec.js

```diff
@@ -431,6 +431,16 @@ describe('src/cy/commands/assertions', () => {
         cy.noop({}).should('dee.eq', {})
       })
 
+      it('throws err when assertion contains only one language chainer', (done) => {
+        cy.on('fail', (err) => {
+          expect(err.message).to.eq('The chainer `be` contains only one language chainer. Please use a valid assertion.')
+
+          done()
+        })
+
+        cy.noop(true).should('be', true)
+      })
```

Comment:
I'd probably update this to go through all the language chainers just to be safe. The test I wrote to check this PR for example: 

```js
// https://github.com/cypress-io/cypress/issues/883
const langChainers = ['to', 'be', 'been', 'is', 'that', 'which', 'and', 'has', 'have', 'with', 'at', 'of', 'same', 'but', 'does', 'still']

langChainers.forEach((langChainer) => {
  it(`throws err when assertion contains only one language chainer: ${langChainer}`, () => {
    cy.noop(true).should(langChainer, true);
  })
})

// combos are not fixed in the PR currently
const langChainerCombos = ['be.and.have.same', 'to.have']

langChainerCombos.forEach((langChainerCombos) => {
  it(`throws err when assertion contains only language chainers: ${langChainerCombos}`, () => {
    cy.noop(true).should(langChainer, true);
  })
})
```

Reply:
Good idea! I've added the suggested tests.